.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "waku/v2/protocol/rln/contracts/rln-contract"]
+	path = waku/v2/protocol/rln/contracts/rln-contract
+	url = git@github.com:vacp2p/rln-contract.git
```

Comment:
err we should use the waku-org/waku-rln-contract now!

Reply:
Using waku-rln-contract `WakuRlnRegistry` introduces a lot of breaking changes, due to how the bindings for go are modified and I end up not being able to retrieve the events:

```go
waku/v2/protocol/rln/group_manager/dynamic/web3.go:17:73: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/web3.go:110:102: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/web3.go:111:27: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/web3.go:172:104: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/web3.go:173:37: gm.rlnContract.FilterMemberRegistered undefined (type *contracts.RLN has no field or method FilterMemberRegistered)
waku/v2/protocol/rln/group_manager/dynamic/dynamic.go:58:59: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/dynamic.go:73:36: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/dynamic.go:76:42: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/dynamic.go:151:24: gm.rlnContract.MEMBERSHIPDEPOSIT undefined (type *contracts.RLN has no field or method MEMBERSHIPDEPOSIT)
waku/v2/protocol/rln/group_manager/dynamic/dynamic.go:231:38: undefined: contracts.RLNMemberRegistered
waku/v2/protocol/rln/group_manager/dynamic/web3.go:173:37: too many errors
```
Notice how I can't easily filter events, nor obtain the fee.
Since @alrevuelta is still using vacp2p/rln-contract, I'll figure out how to solve the errors introduced by changing the contract in a separate PR.